build.zig

```diff
@@ -243,12 +260,35 @@ pub fn build(b: *Build) !void {
         .use_x11 = opt_use_x11,
         .use_egl = opt_use_egl,
         .with_sokol_imgui = opt_with_sokol_imgui,
+        .sokol_imgui_cprefix = opt_sokol_imgui_cprefix,
+        .cimgui_header_path = opt_cimgui_header_path,
+        .imgui_version = imguiver_path,
         .emsdk = emsdk,
+        .use_ubsan = sanitize_c,
+        .use_tsan = sanitize_thread,
     });
+
+    var artifact: ?*std.Build.Step.Compile = null;
```

Comment:
Currently, ldcBuilStep get a library recursive.
If you split the libs, then the optional imgui condition will need to be refactored.